### PR TITLE
Added option to load custom ini (WIP)

### PR DESCRIPTION
**Pull request by alphanu1**
Load a custom ini when using as a library (.so, .dll)

Sorry about the multiple commits.

### CODE REVIEW

File: switchres_wrapper.cpp [hunk 1]
```diff
@@ -34,6 +35,11 @@ MODULE_API void sr_init() {
 	swr->parse_config("switchres.ini");
 }
 
+MODULE_API void sr_load_ini(char config[20]) {
+	swr->parse_config(config+'.'+'i'+'n'+'i');
+
+}
```
Comment (substring) [inline switchres_wrapper.cpp]: Can you:
  * change the parameter to `const char*`
  * remove the forced `.ini` extension
  * remove that empty line

**Review by substring — CHANGES_REQUESTED** [review]
Hi!

Glad to see you here, been watching over your work for years already :wink: Anyway we can get in touch on discord for example ?

For code stuff, see below

**Comment by alphanu1** [comment]
@substring Thanks for the nice welcome. Send me a invite to the channel please 😊

I have sorted these changes.

**Comment by substring** [comment]
And forgot to ask you to squash your commits ;)

For the invitation : https://discord.gg/YtQ6pJh

**Comment by alphanu1** [comment]
@substring, no worries. Will do that shortly

**Comment by alphanu1** [comment]
@substring  all done.

**Comment by substring** [comment]
Reading again, and forgive me as I should have mentioned this at first : the function needs to be in the srAPI struct (in both .h and .cpp).

**Comment by alphanu1** [comment]
@substring not really your fault. To be honest I should have noticed that. Should all be sorted now, hopefully  😊

**Comment by substring** [comment]
So, I finally took the time to test this MR. In the current API state, it only works before `sr_init_disp()`, is that OK with you ?

**Comment by alphanu1** [comment]
Yeah, that should be fine.

**Review by substring — APPROVED** [review]

**Pull request merged**